doc/1-intro/faq.md

```diff
@@ -14,9 +14,34 @@ yes and no. Furnace can be successfully built for Android and it even has some r
 
 nope.
 
+- in order to write any iOS app, the developer must pay a hefty 99 USD per year to just enroll for Apple Developer Program.
+- developing for iOS requires an Apple computer.
+- Furnace's license (GPL) is incompatible with Apple's App Store. since Furnace is free and non-profit, it can be ruled out.
```

Comment:
RetroArch is in the App Store (and that's a GPL-licensed application!). This point is no longer valid.